test/test_framework.py

```diff
@@ -1628,33 +1690,39 @@ def callback_method(self, event: ops.EventBase):
         self.called = True
 
 
-@patch('sys.stderr', new_callable=io.StringIO)
-class BreakpointTests(BaseTestCase):
+@pytest.fixture
+def fake_stderr() -> io.StringIO:
+    return io.StringIO()
```

Comment:
I'm not sure I see the point of this fixture. Can't we just call `io.StringIO` directly if we need to?

Reply:
Yes, it is redundant. I will delete it.